elements/pfe-codeblock/src/pfe-codeblock.js

```diff
@@ -0,0 +1,319 @@
+// Polyfills
+// @POLYFILL  Array.includes
+// https://tc39.github.io/ecma262/#sec-array.prototype.includes
+if (!Array.prototype.includes) {
```

Comment:
Can you move the polyfill into it's own file? `polyfills--pfe-codeblock.js` and import it here?

Reply:
@castastrophe  For some reason when I has the polyfill as a separate file there were issues loading it and I could not figure out why, will give it a go again

Reply:
Feel free to check out pfe-card as an example!

Reply:
I have made the suggested change